.github/workflows/release-desktop-canary.yml

```
@@ -0,0 +1,105 @@
+
+      - name: Delete existing canary release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Delete existing release if it exists
+          gh release delete desktop-canary --yes || true
+          # Delete the tag
+          git push origin :refs/tags/desktop-canary || true
+
+      - name: Create Canary Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)